utils/team_utils.jsx

```diff
@@ -3,22 +3,42 @@
 
 // Use when sorting multiple teams by their `display_name` field
 function compareTeamsByDisplayName(locale, a, b) {
-    if (a.display_name !== b.display_name) {
-        return a.display_name.localeCompare(b.display_name, locale, {numeric: true});
+    if (a.display_name !== null) {
```

Comment:
When would `a.display_name` ever be null? We could also combine this if statement with the following line if it is needed.

Reply:
@hmhealey I added the null check, but I wondered the same thing. If this isn't a realistic scenario I can update the test values and remove the nulls.